backend/base/templates/org_preview2.html

```diff
@@ -1,688 +1,848 @@
-                <!-- Sidebar Content -->
-                <div class="sb-sidenav-menu">
-                    <div class="nav">
-                        <div class="sb-sidenav-menu-heading">Core</div>
-                        <a class="nav-link" href="{% url 'organization_page' %}">
-                            <div class="sb-nav-link-icon"><i class="fas fa-tachometer-alt"></i></div>
-                            Dashboard
-                        </a>
-                        <a class="nav-link" href="{% url 'organization_profile' pk=1 %}">
-                            <div class="sb-nav-link-icon"><i class="fas fa-user fa-fw"></i></div>
-                            Profile
-                        </a>
-                        <a class="nav-link" href="{% url 'organization_locationlist'  %}">
-                            <div class="sb-nav-link-icon"><i class="fas fa-map-marker-alt"></i></div>
-                            Locations
-                        </a>
-                        <a class="nav-link" href="{% url 'preview' %}" style="color: white; background-color: rgb(68, 68, 68);">
-                            <div class="sb-nav-link-icon"><i class="fa-solid fa-list"></i></div>
-                            Preview
-                        </a>
-                        <a class="nav-link" href="{% url 'status' %}">
-                            <div class="sb-nav-link-icon"><i class="fas fa-check-circle"></i></i></div>
-                            Status
-                        </a>
-                        <a class="nav-link" href="{% url 'org_customers_list'%}">
-                            <div class="sb-nav-link-icon"><i class="fas fa-users"></i></div>
-                            Customers
-                        </a>
-                        <div class="sb-sidenav-menu-heading">Slot</div>
-                        <a class="nav-link" href="{% url 'slot-location' %}">
-                            <div class="sb-nav-link-icon"><i class="fa-regular fa-clock"></i></div>
-                            Slots List
-                        </a>
-                        <a class="nav-link" href="{% url 'temp-slot-location'%}">
-                            <div class="sb-nav-link-icon"><i class="fas fa-plus"></i></div>
-                            Additional Slots
-                        </a>
-                        <a class="nav-link" href="{% url 'temp-deslot-location'%}">
-                            <div class="sb-nav-link-icon"><i class="fas fa-minus"></i></div>
-                            Unavailable Slots
-                        </a>
-                    </div>
-                </div>
-                <!-- Logged in User -->
-                <div class="sb-sidenav-footer">
-                    <div class="small">Logged in as:</div>
-                    <div class="user-email">{{ request.user.email }}</div>
-                </div>
-            </nav>
-        </div>
+    <head>

```

Comment:
Are we still having head tag in each HTML pages? Can we have a single page and then extends that too other related pages?

Reply:
Downloaded template from online resources. Template does not support tabs. At the start of development, due to lack of knowledge on JS to Tabs selection and navigation, Instead of using extends tag we did it like this. Will correct it soon.